hedera-node/hedera-schedule-service-impl/build.gradle.kts

```diff
@@ -31,6 +31,8 @@ configurations.all {
 }
 
 dependencies {
+  implementation(project(mapOf("path" to ":hedera-node:hapi-utils")))
+  implementation(project(mapOf("path" to ":hedera-node:hapi-utils")))
```

Comment:
Added twice. And also you can just do 
```
implementation(project(":hedera-node:hapi-utils"))

Reply:
This was auto added by IntelliJ. Thanks for catching it. Fixed.